### PR TITLE
Add Piquasso

### PR DESCRIPTION
**Pull request by Kolarovszki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,7 @@ For a curated list of learning resources please check out [desireevl's repo](htt
 - [OpenQL](https://github.com/QE-Lab/OpenQL) - Compiler framework with algorithm libraries, optimizer, scheduler, QEC, mapping, micro-code generator.
 - [PennyLane](https://pennylane.ai) - Cross-platform Python library for differentiable programming of quantum computers.
 - [Perceval](https://github.com/Quandela/Perceval) - [Quandela](https://www.quandela.com)'s software library for programming realistic photonic quantum computers.
+- [Piquasso](https://github.com/Budapest-Quantum-Computing-Group/piquasso) - A photonic quantum computing simulator library written in Python/C++.
```
Comment (dlyongemallo) [inline README.md:61]: Is this in the correct section? If it's a simulator, it belongs in the simulator section just below. If it's a full stack library, the description should reflect that.

Comment (Kolarovszki) [reply README.md]: Sorry, I moved it to the simulator section.

**Comment by dlyongemallo** [comment]
Thanks @Kolarovszki for the contribution!

**Comment by antalszava** [comment]
Hi @MaldoAlberto, we would love to see Piquasso appear on the list of simulators on the QOSF website for the upcoming mentorship program. I've opened a PR in the [qosf.org repo](https://github.com/qosf/qosf.org/pull/123) too. Could you help with it to reflect the changes on the QOSF website? 🙏

**Comment by MaldoAlberto** [comment]
Hi @antalszava ! Thank you so much for your interest in adding your tool. Right now, we have an issue with our website. Once will is finished, we can take a look and merge your proposal :D

**Pull request merged**